main/keychain/src/main/java/org/cryptomator/keychain/LinuxSystemKeychainAccess.java

```diff
@@ -69,11 +69,11 @@ public LinuxSystemKeychainAccess(Settings settings) {
 
 	/* Getter/Setter */
 
-	public static EnumSet<PwBackend> getAvailablePwBackends() {
-		return availablePwBackends;
+	public static EnumSet<KeychainBackend> getAvailableKeychainBackends() {
+		return availableKeychainBackends;
 	}
 
-	public static PwBackend getBackendActivated() {
+	public static KeychainBackend getBackendActivated() {
```

Comment:
I don't really like this method. `backendActivated` is null unless the constructor is being called. It is kind of a dirty hack to have this method at all.

Is it even used outside of the UI (where it is not even required imho)?